python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2833,6 +2872,7 @@ def _impl(inputs, attr, params, mod):
     "Round": AttrCvt("round"),
     "Rsqrt": _rsqrt(),
     "Select": _where(),
+    "SelectV2": _where(),
```

Comment:
Do we have a test for it?

Reply:
since we are using _where op, just as in the case for "Select", it is the same test as that of _where. Added a new comment in the latest commit that references to SelectV2 test